src/main.cpp

```diff
@@ -963,6 +961,12 @@ int main ( int argc, char** argv )
                 // show dialog
                 ClientDlg.show();
 
+                // Connect on startup
```

Comment:
These shouldn't be in `main.cpp`.

Reply:
I think that's up to discussion. I do understand that one could put this functionality here. How it's handled currently also seems somewhat messy.